src/ocm/components/clusterConfiguration/NetworkConfigurationForm.tsx

```diff
@@ -56,6 +60,23 @@ const NetworkConfigurationForm: React.FC<{
     [hostSubnets, initialValues],
   );
 
+  React.useEffect(() => {
+    if (infraEnvError) {
+      const title = `Failed to retrieve infra env (clusterId: ${cluster.id})`;
+      //TODO(brotman) add handling of existing errors to alerts context
+      if (alerts.find((alert) => alert.title === title)) {
+        return;
+      }
+      captureException(infraEnvError, title);
+      addAlert({
+        title,
+        message: infraEnvError.message,
+      });
+    }
+    //shouldn't respond to cluster polling. shouldn't respond to alerts changes so remove alert wouldn't trigger adding it back
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [infraEnvError, addAlert]);
```

Comment:
Actually, I think it should react only to a new `infraEnvError`.
WDYT?
```suggestion
  }, [infraEnvError]);
```